.github/workflows/release-grimoirelab-component.yml

```diff
@@ -103,12 +103,14 @@ jobs:
         name: Check if package dependencies exist
         run: |
           dependencies="${{ inputs.dependencies }}"
+          timeout=5
```

Comment:
This variable shouldn't be defined after the `then` clause?

Reply:
Yes, you are right!